src/main/java/uk/gov/hmcts/reform/sscs/controller/SyaController.java

```diff
@@ -43,4 +47,20 @@
 
         return status(201).build();
```

Comment:
Should this not also return the http header `location`?  Use `HttpStatus.CREATED` instead of `201`.